coding-style.md

```diff
@@ -33,7 +33,7 @@ Think of the leading underscore as a shorthand for `this->`.
 Template parameters use `CamelCase`
 
 Note: because the Concept Technical Specification used CamelCase for concepts,
-some Seastar concepts also use CamelCase. These will be gradually deprecated
+some Seastar concepts also uses CamelCase. These will be gradually deprecated
```

Comment:
This change is incorrect English - we really need the plural "use" here, not "uses".

Reply:
You're right... It was manual, which means that's purely on me. I must have been tired. I think I must have parsed it as <Seastar> also uses ..., but you're right. 🤦